packages/cli/src/Server.ts

```diff
@@ -481,7 +481,7 @@ export class Server extends AbstractServer {
 				logger,
 				jwtService,
 			}),
-			new TagsController({ config, repositories, externalHooks }),
+			new TagsController({ config, tagRepository: Container.get(TagRepository), externalHooks }),
```

Comment:
maybe we can import `config` directly in `TagsController`, add `@Service()` to that class, and replace this line with `Container.get(TagsController)`. 
That's the eventual goal: to stop explicitly passing in dependencies everywhere.